html/elements/img.json

```
@@ -185,10 +185,14 @@
                 "version_added": "57"
               },
               "safari": {
-                "version_added": "14"
+                "version_added": "14",
+                "partial_implementation": true,
+                "notes": "Safari's implementation does not yet work properly with lazy-loaded images. See: https://webkit.org/b/224197"
```

I misunderstood this note at first to mean that it somehow prevents lazy loading. How about we revise it somewhat to make it clear that this affects layout. What do you think of this?

```suggestion
                "notes": "Safari doesn't preserve space for images without a valid <code>src</code>, which may disrupt layouts that rely on lazy loading (see <a href='https://webkit.org/b/224197'>bug 224197</a>)."
```